src/tests/sqlsmith/tests/test_runner.rs

```diff
@@ -19,47 +19,57 @@ use std::{env, panic};
 
 use rand::SeedableRng;
 use risingwave_frontend::binder::Binder;
-use risingwave_frontend::handler::create_table;
 use risingwave_frontend::planner::Planner;
 use risingwave_frontend::session::{OptimizerContext, OptimizerContextRef, SessionImpl};
 use risingwave_frontend::test_utils::LocalFrontend;
-use risingwave_frontend::FrontendOpts;
+use risingwave_frontend::{handler, FrontendOpts};
 use risingwave_sqlparser::ast::Statement;
 use risingwave_sqlparser::parser::Parser;
-use risingwave_sqlsmith::{sql_gen, Table};
+use risingwave_sqlsmith::{create_mview_sql_gen, sql_gen, Table};
 
 /// Create the tables defined in testdata.
 async fn create_tables(session: Arc<SessionImpl>) -> Vec<Table> {
     let sql = std::fs::read_to_string("tests/testdata/tpch.sql").unwrap();
     let statements =
         Parser::parse_sql(&sql).unwrap_or_else(|_| panic!("Failed to parse SQL: {}", sql));
+    let n_statements = statements.len();
 
     let mut tables = vec![];
     for s in statements.into_iter() {
         match s {
             Statement::CreateTable {
-                name,
-                columns,
-                with_options,
+                ref name,
+                ref columns,
                 ..
             } => {
-                let context = OptimizerContext::new(session.clone(), Arc::from(sql.clone()));
-                create_table::handle_create_table(
-                    context,
-                    name.clone(),
-                    columns.clone(),
-                    with_options,
-                )
-                .await
-                .unwrap();
-                tables.push(Table {
-                    name: name.0[0].value.clone(),
-                    columns: columns.iter().map(|c| c.clone().into()).collect(),
-                })
+                let name = name.0[0].value.clone();
+                let columns = columns.iter().map(|c| c.clone().into()).collect();
+                handler::handle(session.clone(), s, &sql).await.unwrap();
+                tables.push(Table { name, columns })
             }
             _ => panic!("Unexpected statement: {}", s),
         }
     }
+
+    let mut rng = rand::thread_rng();
+    let mut sql_generator = create_mview_sql_gen(&mut rng, tables.clone());
+
+    // Generate Materialized Views 1:1 with tables, so they have equal weight
+    // of being queried.
+    for i in 0..n_statements {
+        let (stmt, columns) = sql_generator.gen_mview(&format!("m{}", i));
+        match stmt {
```

Comment:
Column names are in wrong format. They should not be qualified, otherwise look up fails.

Reply:
Edit: sql gen generates strings, in that context column names be qualified as strings.